Bottom: 0690fcc72ee353404be62a31e547f5e459f5eb4a
Top:    3f0f7b375595e236d26326d6ffa3258761669605
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 23:14:42 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
